server/api/users/update.js

```diff
@@ -0,0 +1,40 @@
+const { responses } = require('../../responses')
+const database = require('../../database')
+const config = require('../../../config')
+
+module.exports = {
+  method: 'patch',
+  path: '/users/me',
+  requireAuth: true,
+  schema: {
+    body: {
+      type: 'object',
+      properties: {
+        name: {
+          type: 'string'
+        },
+        division: {
+          type: 'number',
+          enum: Object.values(config.divisions)
+        }
+      }
+    }
+  },
+  handler: async ({ uuid, req }) => {
+    const { name, division } = req.body
+
+    const user = await database.auth.updateUser({
+      id: uuid,
+      name,
+      division
+    })
+
+    return [responses.goodUserUpdate, {
```

Comment:
Is it necessary to return user data here? This seems like a place we could forget to update if we ever add a column to the user table.

Reply:
The data returning is merely meant as a convenience for testing, I don't think the client would ever rely on it? The client can always get their data from `/users/me`